interpreter/cling/lib/Interpreter/CIFactory.cpp

```diff
@@ -1252,6 +1253,55 @@ namespace {
     }
   }
 
+  // Goal is to make this as close to the function in
+  // clang/lib/Interpreter/Interpreter.cpp taking only clang arguments
+  static llvm::Expected<std::unique_ptr<CompilerInstance>>
```

Comment:
I think this function never returns an error. Instead it prints directly to `cling::errs()` and returns a "successful" `nullptr`...